drivers/at/at.c

```diff
@@ -54,18 +54,23 @@ int at_expect_bytes(at_dev_t *dev, const char *bytes, size_t len, uint32_t timeo
     return 0;
 }
 
+void at_send_bytes(at_dev_t *dev, const char *bytes, size_t len)
+{
+    uart_write(dev->uart, (const uint8_t *)bytes, len);
+}
+
 int at_send_cmd(at_dev_t *dev, const char *command, uint32_t timeout)
 {
     unsigned cmdlen = strlen(command);
 
     uart_write(dev->uart, (const uint8_t *)command, cmdlen);
-    uart_write(dev->uart, (const uint8_t *)"\n", 1);
+    uart_write(dev->uart, (const uint8_t *)AT_END_OF_LINE, sizeof(AT_END_OF_LINE) - 1);
 
     if (at_expect_bytes(dev, command, cmdlen, timeout)) {
         return -1;
     }
 
-    if (at_expect_bytes(dev, "\r\n", 2, timeout)) {
+    if (at_expect_bytes(dev, AT_END_OF_LINE "\r\n", sizeof(AT_END_OF_LINE) + 1, timeout)) {
```

Comment:
When you send `AT<CR>`, the modem answer `AT<CR>` (echo) then `<CR><LF>OK`. I guess this work like that for every modem...maybe this could be configurable as well, not sure if needed though.